### DISCUSSION TITLE
bug Updated app won't connect

### DISCUSSION THREAD

**Discussion by qcks** [immich-app/immich#13751] [Q&A] [answered]
### The bug

ApiException 400: Socket operation failed: GET /server/ping (Inner exception: ClientException with SocketException: Connection failed (OS Error: No route to host, errno = 65), address = 192.168.1.41, port = 2283, uri=http://192.168.1.41:2283/api/server/ping)

### The OS that Immich Server is running on

macOS，v1.115.0

### Version of Immich Server

v1.115.0

### Version of Immich Mobile App

v1.118.0

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
irrelevant
```

### Your .env content

```Shell
irrelevant
```

### Reproduction steps

irrelevant

### Relevant log output

```shell
Updated app won't connect，Updated app won't connect
```

### Additional information

Updated app won't connect

**Comment by danieldietzler** [comment, marked as answer]
Update your server.

**Reply by sedemamediku** [reply]
my server is 1.118.2 and my mobile app is 1.118.0 but i still cannot connect

**Reply by danieldietzler** [reply]
@kofinco don't hijack an existing discussion.

**Reply by kofixsa** [reply]
> @kofinco don't hijack an existing discussion.

sorry i had the same issue so i thought i could ask here. my bad
